
From: Christoph Hellwig <hch@lst.de>

Besides namei.c it's only used in the SN2 hwgraph code which can't be
modular (and will be removed soon)

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/namei.c |    1 -
 1 files changed, 1 deletion(-)

diff -puN fs/namei.c~unexport-lookup_create fs/namei.c
--- 25/fs/namei.c~unexport-lookup_create	2004-10-03 16:43:22.750872896 -0700
+++ 25-akpm/fs/namei.c	2004-10-03 16:43:22.754872288 -0700
@@ -2386,7 +2386,6 @@ EXPORT_SYMBOL(follow_up);
 EXPORT_SYMBOL(get_write_access); /* binfmt_aout */
 EXPORT_SYMBOL(getname);
 EXPORT_SYMBOL(lock_rename);
-EXPORT_SYMBOL(lookup_create);
 EXPORT_SYMBOL(lookup_hash);
 EXPORT_SYMBOL(lookup_one_len);
 EXPORT_SYMBOL(page_follow_link);
_
